src/components/ha-attributes.ts

```diff
@@ -116,19 +125,38 @@ class HaAttributes extends LitElement {
       const yaml = jsYamlPromise.then((jsYaml) => jsYaml.safeDump(value));
       return html` <pre>${until(yaml, "")}</pre> `;
     }
-    // URL handling
-    if (typeof value === "string" && value.startsWith("http")) {
-      try {
-        // If invalid URL, exception will be raised
-        const url = new URL(value);
-        if (url.protocol === "http:" || url.protocol === "https:")
-          return html`<a target="_blank" rel="noreferrer" href="${value}"
-            >${value}</a
-          >`;
-      } catch (_) {
-        // Nothing to do here
+
+    if (typeof value === "string") {
+      // URL handling
+      if (value.startsWith("http")) {
+        try {
+          // If invalid URL, exception will be raised
+          const url = new URL(value);
+          if (url.protocol === "http:" || url.protocol === "https:")
+            return html`<a target="_blank" rel="noreferrer" href="${value}"
+              >${value}</a
+            >`;
+        } catch (_) {
+          // Nothing to do here
+        }
+      }
+      // Timestamp handling
+      if (isTimestamp(value)) {
+        const date = new Date(value);
+        if (checkValidDate(date)) {
+          return formatDateTimeWithSeconds(date, this.hass.locale);
+        }
+      }
+
+      // Date handling
+      if (isDate(value)) {
```

Comment:
Could be potentially optimized by first checking if the string starts with a date and if that is the case and there are more characters left after it then validate against a timestamp. But not sure if worth the effort and complexity.

Reply:
If `isDate` also matches a timestamp, you should move the timestamp check in here.

Reply:
I adjusted the RegExp and their order for the discussed runtime efficiency.